### DISCUSSION TITLE
After update Victron 3 the driver 14.3 is not working

### DISCUSSION THREAD

**Discussion by rbeudel** [Louisvdw/dbus-serialbattery#679] [Q&A] [answered]
Hello,
I updatet the venusos on my raspi 3 to the Version 3. After that I became the driver not working. I deinstalled the driver and installed new, but it is not running. This is last part of "current"
```bash
@4000000064771a19017901ac serstart starting
@4000000064771a1902177fe4 INFO: loading config file /etc/venus/serial-starter.conf
@4000000064771a2c2afdf564 INFO: loading config file /data/conf/serial-starter.d
@4000000064771a2e149e2ce4 INFO: Create daemontools service dbus-cgwacs.ttyUSB0
@4000000064771a2f3438474c INFO: Create daemontools service mk2-dbus.ttyUSB1
@4000000064771a3421dc5f84 INFO: Start service dbus-cgwacs.ttyUSB0 once
@4000000064771a360860a934 INFO: Start service mk2-dbus.ttyUSB1
```

**Comment by mr-manuel** [comment]
Did you check the [documentation](https://louisvdw.github.io/dbus-serialbattery/) and [How to troubleshoot](https://louisvdw.github.io/dbus-serialbattery/troubleshoot/)?

If not, please read the documentation. If yes please read the documentation and provide correct logs.

**Comment by rbeudel** [comment, marked as answer]
Did you check... Of cource
I roll back to 2.94 today. To provide correct logs I updated again to 3.0 and wondering the serial driver is working, I dont know why. Thank you for your responce and the good work.
Ralf

**Reply by mr-manuel** [reply]
For the logs please execute always the full command, e.g.

```bash
tail -f /data/log/serial-starter/current | tai64nlocal
```

else the datetime is not converted in a readable format that we need to troubleshoot.

So now it works for you in Venus OS `v3.00` and driver `v1.0.20230531`?

**Reply by rbeudel** [reply]
I checked in the moment. It is Venus OS v3.00 and displayed under battery firmwareversion 14.3.  I will try the new version next days. Thank you very much.

**Reply by mr-manuel** [reply]
Even when you check in the moment, it can change a lot, if between one message and the other are 2 seconds, 10 seconds or 30 seconds. In the log you postet this information is "lost".

**Reply by rbeudel** [reply]
I understand. Next time I will do it better. But I hope, I will have no next time.
